### PR TITLE
Add AWS Lambda function to create users in Cognito

### PR DESCRIPTION
**Pull request by vladdy**
This is the main work horse for acl-org/acl-2020-virtual-conference/issues/55

### CODE REVIEW

File: acl2020_tools/awscognito/create_user_lambda.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#  AmazonCognitoPowerUser - yes, not that secure, but works for test  
+# 
+# Runtime:
+#   Python 3.7                                  
+
+import os
+import json
+import boto3
+from dataclasses import dataclass
+import cognito  # type: ignore
```
Comment (hao-fang) [inline acl2020_tools/awscognito/create_user_lambda.py:22]: this means you need to upload this `cognito.py` file to the Lambda function, right?
Did you check if it works properly? (I guess so, just want to make sure there is no import issue..)

**Review by hao-fang — COMMENTED** [review]
You will need to run `make format` locally to make Github Action happy :)

File: acl2020_tools/awscognito/create_user_lambda.py (hunk 1 above)
Comment (vladdy) [reply acl2020_tools/awscognito/create_user_lambda.py:22]: Yes for the both questions :)

**Comment by vladdy** [comment]
@hao-fang can you please take another look at this when you have time?

**Review by hao-fang — APPROVED** [review]
@vladdy Thanks! Sorry the first few days are crazy but things are getting better :)

**Pull request merged**